lms/static/sass/discussion/utilities/_shame.scss

```diff
@@ -29,6 +29,19 @@
   position: relative;
 }
 
+// Temporary breadcrumbs
+.has-breadcrumbs {
```

Comment:
Could we use the real Pattern Library breadcrumb styling, even if the rendering doesn't show links? That would avoid a little bit more tech debt, if it is possible.